### PR TITLE
Improve presentation of Harvester Manager Built-in Extension

### LINKED ISSUE #7995: Presentation issues on the Extensions page
- The icon beneath the preview panel shows through - assume this is a z-index issue
- Harvester Manager build-in extension needs an icon and a description. Name should probably be 'Virtualization Management'

### PR DESCRIPTION
**Pull request by nwmac**
Fixes #7995 

- Fixes the icon overlap issue
- Adds support for built-in extensions to override the display label
- Fixed info panel to use the display label not name
- Improved display of version number for built-in plugins to be consistent with others
- Updated Harvester Manager extension to have better name, description and icon

Extensions view:
image

Info panel:
image

### CODE REVIEW

File: pkg/harvester-manager/package.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "name": "harvester-manager",
-  "description": "harvester-manager plugin",
+  "description": "Provides Virtualization Management feature in Rancher Manager",
```
Comment (mantis-toboggan-md) [inline pkg/harvester-manager/package.json:3]: There should probably be a "the" before Virtualization

**Review by mantis-toboggan-md — CHANGES_REQUESTED** [review]
This isn't part of the original issue but I'm complaining about it here anyway. I noticed that the extensions page switches to the 'available' tab if no extensions are installed, but that check appears to ignore the harvester manager plugin. I think it should default to the 'installed' tab if there are any plugins listed there regardless of whether they're built in or installed.

Aside from that and my comment about the description this PR looks good.

**Comment by krumware** [comment]
Just a quick note to bring attention if relevant, harvester-manager imports currently break the extension starter
https://github.com/rancher/dashboard/issues/8008

**Comment by nwmac** [comment]
@mantis-toboggan-md Fixed description - I'll create an issue for considering built-in extensions when making initial tab choice.

**Review by mantis-toboggan-md — APPROVED** [review]

**Comment by nwmac** [comment]
@krumware Thanks - a fix for this is imminent

**Pull request merged**